mapclassify/classifiers.py

```diff
@@ -1123,6 +1123,68 @@ def plot(
             plt.savefig(file_name, dpi=dpi)
         return f, ax
 
+    def plot_histogram(
+        self,
+        hist_color="dodgerblue",
```

Comment:
Shall we just simply call this `color`?

Reply:
makes sense to me. Wasnt sure if we should differentiate between the bar color and the line color. Seems reasonable that 'color' defaults to the bar color